### DISCUSSION TITLE
How do I use pnpm?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [stackblitz/core#1907] [Q&A]
I'd like to use pnpm instead of npm for a variety of reasons

npm v7 doesn't even have overrides/resolutions, and I need to test some PRs (e.g.: https://github.com/broccolijs/broccoli/pull/499)

pnpm also has a correct peer dependencies implementation (and is very fast)

**Comment by kc0tlh** [comment]
This is now supported in [our new VS Code editor](https://stackblitz.com/codeflow)!
